localization/english/culture/cultures_l_english.yml

```diff
@@ -374,8 +374,8 @@
  croatian_collective_noun:1 "Hrvatski"
  serbian:0 "Srpski"
  serbian_collective_noun:1 "Srbi"
- vlach:0 "Român"
- vlach_collective_noun:1 "Român"
+ vlach:0 "Rumân" # Later transformed into "Român" as "rumân" eventually became a term exclusive for the lower class
+ vlach_collective_noun:1 "Rumân"
```

Comment:
It should be plural - won't it be Rumăni?

Reply:
Rumâni*

Or "Rumîni" as someone suggested on the sheets. I've added to the discussion there